test/Table_Tests/src/IO/Excel_Spec.enso

```diff
@@ -318,6 +320,19 @@ spec_write suffix test_sheet_name =
             written.should_equal expected
             out.delete_if_exists
 
+        Test.specify 'should be able to write to a new dry run file' <|
```

Comment:
Also a test for an existing but empty file?

Reply:
+1